manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+# tested with a clean 64 bit installation of Raspbian-Lite
+
+function install_core (){
+    echo "Installing OVOS core"
+    echo
+    pip3 install -U pip
+
+    # install ovos-core - ovos-backend-client has conflicting dependencies
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-backend-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-core
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-audio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
+    pip3 install tornado
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-messagebus
+    pip3 install adapt-parser
+    pip3 install padacioso
```

Comment:
adapt and padacioso not needed, it is installed by ovos-core

Reply:
Gone from my copy. Will test.